template_helpers/views.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from django.views.generic import TemplateView
+
+
+class GenericTemplateView(TemplateView):
+
+    def get_template_names(self):
+        template_filename = '%s.html' % (self.kwargs['template'] or 'index')
+        return [os.path.join('tests', template_filename)]
```

Comment:
The base directory ("tests" should be changeable via as_view kwarg. We might also add a "get_base_directory()" method to allow further overriding the default behaviour.